library.properties

```diff
@@ -1,5 +1,5 @@
 name=Octoliner
-version=1.2.2
+version=1.2.5
```

Comment:
I think we should not increase the patch version after each commit. It will be enough to increase it by 1 after the merger.

Reply:
As I understand Victor's words: "the version changes with any change in the code, not comments, not auxiliary files, but the code". Perhaps I misunderstood, let him guide us on the right path.

Reply:
@nkrkv Please tell us your opinion.

Reply:
The `version` and semver is all about _releasing_. The version has to be bumped when you are about releasing a new version, perform deployment, etc. Albeit there’s nothing wrong with bumping a version in every commit, it’s hard to manage, easy to forget, and easy to get a conflict when multiple developers change the same code base addressing different issues. So, in other projects, we bump right before releasing, so do many other projects I know, and it causes no problems.

To put in summary:

- The @yurybotov change is OK
- It was not strictly necessary